.github/workflows/integrationTest.yml

```diff
@@ -667,10 +667,8 @@ jobs:
               -var="binary_name=amazon-cloudwatch-agent.rpm" \
               -var="local_stack_host_name=${{ needs.StartLocalStack.outputs.local_stack_host_name }}" \
               -var="s3_bucket=${S3_INTEGRATION_BUCKET}" \
-              -var="vpc_security_group_ids=${VPC_SECURITY_GROUPS_IDS}" \
               -var="key_name=${KEY_NAME}" \
               -var="test_name=cw-integ-test-al2" \
-              -var="iam_instance_profile=${IAM_ROLE}" \
```

Comment:
I thought this was already updated in a previous pr. Did I miss this?

Reply:
Maybe the `iam_instance_profile` that we thought was in integration test. This is for the performance tracking test after the merge which I forgot to update it.